api/osrd_infra/views/infra.py

```diff
@@ -30,7 +31,10 @@ def perform_create(self, serializer):
 
     @action(url_path="railjson", detail=True, methods=["get"])
     def get_railjson(self, request, pk=None):
-        railjson = serialize_infra(self.get_object())
+        exclude_extensions = request.query_params.get("exclude_extensions", "false").lower() == "true"
+        cache_key = f"osrd.infra.{pk}.{exclude_extensions}"
+        railjson = cache.get(cache_key)
```

Comment:
I think you copied an old version the code.
Back then we were using redis to cache railjson infra cause we had performance issues generating railjson. This is not the case anymore which make cache usage overkill.